test/system/books_test.rb

```diff
@@ -3,45 +3,68 @@
 require 'application_system_test_case'
 
 class BooksTest < ApplicationSystemTestCase
+  include Devise::Test::IntegrationHelpers
+
   setup do
-    @book = books(:one)
+    sign_in users(:one)
   end
 
   test 'visiting the index' do
     visit books_url
-    assert_selector 'h1', text: 'Books'
+    assert_selector 'h1', text: '本'
   end
 
-  test 'creating a Book' do
+  test 'creating a book' do
     visit books_url
-    click_on 'New Book'
+    click_on '新規作成'
+
+    fill_in 'book[title]', with: 'The Return of the King'
+    fill_in 'book[memo]', with: 'final volume'
+    click_on '登録する'
 
-    fill_in 'Memo', with: @book.memo
-    fill_in 'Title', with: @book.title
-    click_on 'Create Book'
+    assert_text '本が作成されました。'
 
-    assert_text 'Book was successfully created'
-    click_on 'Back'
+    assert_text 'The Return of the King'
+    assert_text 'final volume'
   end
 
-  test 'updating a Book' do
+  test 'updating a book' do
     visit books_url
-    click_on 'Edit', match: :first
 
-    fill_in 'Memo', with: @book.memo
-    fill_in 'Title', with: @book.title
-    click_on 'Update Book'
+    click_on '編集', match: :prefer_exact
+
+    assert_selector :field, type: 'text', with: 'The Fellowship of the Ring'
```

Comment:
```suggestion
    assert_field 'タイトル', with: 'The Fellowship of the Ring'
```

みたいに書いた方がシンプルになりそうです